src/css/base/_form.scss

```diff
@@ -0,0 +1,75 @@
+@use 'sass:math';
+
+/* Form */
+select,
+textarea,
+input[type='text'],
+input[type='password'],
+input[type='datetime'],
+input[type='datetime-local'],
+input[type='date'],
+input[type='month'],
+input[type='time'],
+input[type='week'],
+input[type='number'],
+input[type='email'],
+input[type='url'],
+input[type='search'],
+input[type='tel'],
+input[type='color'] {
+    @extend .form-control;
+
+    &.error {
+        @extend .is-invalid;
+    }
+}
+
+.formbody {
```

Comment:
Those look Contao specific. I don't think they are supposed to be placed in OpenFrontend.